dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -349,5 +351,28 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
         application.send_decision_notification_email
       end
     end
+
+    test 'accepted_at updates times' do
+      time_1 = Date.today.to_time
+      time_2 = Date.tomorrow.to_time
```

Comment:
nit: how about `today` and `tomorrow` instead of `time_1` and `time_2` for clarity?